app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,9 @@
       { text: "Log out", href: gds_sign_out_path }
     ]
   }%>
+<%= javascript_include_tag "https://www.gstatic.com/charts/loader.js" %>
+<%= javascript_include_tag "application" %>
+<%= stylesheet_link_tag "application" %>
```

Comment:
From govuk_publishing_components v9.13 it is possible to move this in the `<head>` of the html